sdk/export/trace/tracetest/test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tracetest // import "go.opentelemetry.io/otel/sdk/export/trace"
+
+import (
+	"context"
+	"sync"
+
+	"go.opentelemetry.io/otel/sdk/export/trace"
+)
+
+// NewNoopSpanBatcher returns a new no-op trace.SpanBatcher.
+func NewNoopSpanBatcher() trace.SpanBatcher {
+	return new(noopSpanBatcher)
+}
+
+type noopSpanBatcher struct {
+}
+
```

Comment:
Can we add a compile time assertion that this actually implements the interface(s) it intends to?

```suggestion
var _ trace.SpanBatcher = (*noopSpanBatcher)(nil)
```

Reply:
Gotcha, I was anticipating the SpanSyncer interface also be implemented.

That said, there is recent thought to revise these interfaces into a single interface. We could forego these changes with the anticipation of that refactor. Thoughts?

Reply:
Done, now that implement both interfaces.